libraries/state_history/abi.cpp

```diff
@@ -33,6 +33,19 @@ extern const char* const state_history_plugin_abi = R"({
                 { "name": "fetch_deltas", "type": "bool" }
             ]
         },
+        {
+            "name": "get_blocks_request_v1", "fields": [
```

Comment:
just curious, why did we need a `_v1` for `get_blocks_request_v1`, but not for `get_blocks_result_v0`?

Reply:
We need a `_v1` for request as an existing client has to add `fetch_finality_data` in a request if we put `fetch_finality_data` in the ABI definition. But we don't need a new version for `get_blocks_result` since we only add the finality_data log in new Savanna.

Reply:
You might be able to make `fetch_finality_data` a binary extension on `get_blocks_request_v0` to avoid needing a v1. Whether or not that's a good idea I'm not sure

Reply:
Seems like we should just keep it simple and straightforward. Binary extension seems overly complex.